contracts/src/v0.8/dev/ocr2dr/OCR2DRClient.sol

```diff
@@ -20,6 +20,14 @@ abstract contract OCR2DRClient is OCR2DRClientInterface {
   error RequestIsAlreadyPending();
   error RequestIsNotPending();
 
+  constructor(address oracle) {
+    setOracle(oracle);
+  }
+
+  function getDONPublicKey() external view override returns (bytes32) {
```

Comment:
Add Natspec comment for this.  I think that you should be able to just inherit the comment from the interface.

```
/// @inheritdoc IOCR2DRClientInterface
```